requests/request.go

```diff
@@ -19,6 +24,11 @@ type Auth struct {
 	Secret string
 }
 
+type FileUploadParams struct {
+	File   *os.File
+	Fields map[string]string
```

Comment:
change this to map[string] interface

Reply:
I have tried, but interface type is not supporting in this function second parameter  `writer.WriteField` 
```
for fieldName, fieldValue := range params.Fields {
    err = writer.WriteField(fieldName, fieldValue)
       if err != nil {
	log.Fatal(err)
    }
}
```
https://github.com/razorpay/razorpay-go/blob/add_partner_api/requests/request.go#L253

Reply:
@ndhaka007 I have tried, but interface type is not supporting in this function second parameter  `writer.WriteField` 
```
for fieldName, fieldValue := range params.Fields {
    err = writer.WriteField(fieldName, fieldValue)
       if err != nil {
	log.Fatal(err)
    }
}
```
https://github.com/razorpay/razorpay-go/blob/add_partner_api/requests/request.go#L253